vms/job/templates/job/create.html

```diff
@@ -28,21 +28,34 @@
                             Event:
                         </label>
                         <div class="col-md-8">
-                            <select class="form-control" name="event_id" id="id" onchange="dates()">
+                            <select class="form-control" name="event_id" id="events" onchange="update_event_dates()">
                                 {% for event in event_list %}
-                                    <option value="{{ event.id }}" event="{{ event }}">{{ event.name }}</option>
+                                    <option value="{{ event.id }}" start_date="{{ event.start_date }}" end_date="{{ event.end_date }}">{{ event.name }}</option>
                                 {% endfor %}
                             </select>
                         </div>
                     </div>
-                    <p id="d"></p>
-                    <script type="text/javascript" src="{{ STATIC_URL }}js/jquery-1.8.3.min.js">
-                       function dates(){
-                            var event_id=document.getElementById("id");
-                            event=event_id.getAttribute("event");
-                            document.getElementById("d").innerHTML="Event starts by {{ event.start_date }} and ends by {{ event.end_date }}";
-                    </script>
-
+					<div class="form-group">
+						<label class="col-md-2 control-label">Event Start Date</label>
+						<div class="col-md-10">
+							<p class="form-control-static" id="start_date_here">start_date</p>
+						</div>
+						<label class="col-md-2 control-label">Event End Date</label>
+						<div class="col-md-10">
+							<p class="form-control-static" id="end_date_here">end_date</p>
+						</div>
+					</div>
+                    <script type="text/javascript">
+						function update_event_dates(){
```

Comment:
@ahmedsabie Please separate the JS code. Just look if this can be put into some already existing JS file or add a new JS file and put this into that, this will make things more clean and clear.